backend/Dockerfile

```diff
@@ -1,18 +1,22 @@
-FROM ubuntu:16.04
+FROM python:3.8
 
 EXPOSE 5000
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
```

Comment:
do we need these lines if we're migrating from `ubuntu:16.04` to `python:3.8`?

Reply:
Removed.